vscode/package.json

```diff
@@ -142,14 +158,13 @@
       {
         "command": "qsharp-vscode.downloadResults",
         "category": "Q#",
-        "title": "Download full Azure Quantum job results",
+        "title": "Download Azure Quantum job results",
         "icon": "$(cloud-download)"
       },
       {
         "command": "qsharp-vscode.setTargetProfile",
         "category": "Q#",
-        "title": "Set the Azure Quantum QIR target profile",
-        "enablement": "resourceLangId == qsharp"
+        "title": "Set the Azure Quantum QIR target profile"
```

Comment:
Why did we remove the enablement filter on this?

Reply:
It's a global setting (saved in your user settings), not document specific. I'm not sure you should have to have a Q# file open to be able to change it from the menu. I could be persuaded either way though.